integration/configs/video_test_processor.config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "`": 1,
+  "name": "video_test_processor.config.json %ts",
+  "grids": [
+  ],
+  "tests": [
+    {
+      "name": "Video Test Processor %ts",
+      "description": "Video Test Processor",
+      "tupleSize": 2,
+      "noOfThreads": 1,
+      "testImpl": "VideoTest.js",
+      "payload": {
+        "url": "http://localhost:8080/",
+        "port": 30015,
+        "retry": 2,
+        "useVideoProcessor": true
+      }
+    }
+  ],
+  "clients": [
+    {
+      "browserName": "chrome",
+      "version": "latest",
+      "platform": "MAC"
+    },
+    {
+      "browserName": "firefox",
```

Comment:
Dont use FF for video test for now. It fails intermittently

Reply:
sounds good